episodes/05-creating-histograms.md

```diff
@@ -411,9 +410,9 @@ mask[circle] = 1
 
 # just for display:
 # make a copy of the image, call it masked_image, and
-# use np.logical_not() and indexing to apply the mask to it
-masked_img = wellplate[:]
-masked_img[np.logical_not(mask)] = 0
+# zero values where mask is False
```

Comment:
I think this is slightly less confusing than the existing solution and dosnt introduce a new way to copy an array.

Reply:
Ill change to `masked_img[~mask] = 0` which is consistent with exercises in the thresholding episode